ros_ws/src/autonomous/q_learning/scripts/plotter.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+
+from PyQt4 import QtCore, QtGui
+from parameters import *
+import pyqtgraph as pg
```

Comment:
Please add a note to the readme on how to install pyqt4 and pyqtgraph.
I just installed them like this:
```
sudo apt install python-qt4
sudo python -m pip install pyqtgraph
```

Reply:
I've already added `python-pyqtgraph` as an apt package which also includes qt4.